test/sanity/validate-modules/ignore.txt

```diff
@@ -198,6 +198,8 @@ lib/ansible/modules/cloud/azure/azure_rm_postgresqlserver.py E326
 lib/ansible/modules/cloud/azure/azure_rm_publicipaddress.py E322
 lib/ansible/modules/cloud/azure/azure_rm_publicipaddress.py E324
 lib/ansible/modules/cloud/azure/azure_rm_publicipaddress_facts.py E324
+lib/ansible/modules/cloud/azure/azure_rm_resource.py E324
+lib/ansible/modules/cloud/azure/azure_rm_resource_facts.py E324
```

Comment:
New ignores should not be added. The issues should be fixed instead. If the errors are due to module_utils or docs fragments, those should be fixed in a separate PR before merging this one.

Reply:
ok, so i will create that separate pr then